src/support/generation.c

```diff
@@ -109,6 +109,8 @@ __gen_drain_callback(
     WT_CONNECTION_IMPL *conn;
     WT_GENERATION_DRAIN_COOKIE *cookie;
     uint64_t time_diff_ms, v;
+    WT_VERBOSE_LEVEL verbose_tmp[WT_VERB_NUM_CATEGORIES];
```

Comment:
Since `verbose_tmp` is only modified inside `#ifdef HAVE_DIAGNOSTIC` we'll get an unused warning when compiling in release mode. I'd also recommend a slight tweak to the name to avoid using tmp
```suggestion
#ifdef HAVE_DIAGNOSTIC
    WT_VERBOSE_LEVEL verbose_orig_level[WT_VERB_NUM_CATEGORIES];
#endif
```